Examples/options/LCIO_read_G2CD.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+from Gaudi.Configuration import *
+from Configurables import K4DataSvc
+dsvc = K4DataSvc("EventDataSvc")
```

Comment:
Please also rename it to k4.

Reply:
Hi @mirguest ，

Thanks for the comments, I have updated the pull request.

Best,
Wenxing